src/golang/lib/repos/user.go

```diff
@@ -16,21 +16,17 @@ type User interface {
 
 type userReader interface {
 	// GetByAPIKey returns the User with the API key apiKey.
+	// It returns a database.ErrNoRows if no rows are found.
 	GetByAPIKey(ctx context.Context, apiKey string, DB database.Database) (*models.User, error)
-
-	// GetOrgAdmin returns the admin User for the organization orgID.
-	GetOrgAdmin(ctx context.Context, orgID string, DB database.Database) (*models.User, error)
 }
 
 type userWriter interface {
 	// Creates inserts a new User with the specified fields.
 	Create(
 		ctx context.Context,
-		email string,
 		orgID string,
-		role string,
 		apiKey string,
-		auth0ID string,
```

Comment:
Why did we make these changes?

Reply:
Similarly, the higher level call to this was just passing in an empty string for all these columns. We can add back these parameters if and when we have multiple users and start using these fields.